start-from-zero/Javascript/src/views/evaluation/evaluation.js

```diff
@@ -0,0 +1,122 @@
+function EvaluationsPage(options){
+    const headings = ['Name','Language','Level',''];
+    
+    const rows = [
+        {
+            name:'Dan Constantinescu',
+            language: 'Javascript',
+            level:'Junior'
+        },
+        {
+            name:'Adolf Hgg',
+            language: 'SwS',
+            level:'Master'
+        },
+        {
+            name:'Fiodor D',
+            language: 'Russian',
+            level:'Master'
+        },
+    ];
+
+    const newOptions = options || {};
+    return `
+    ${NAV()}
+    ${EvaluationsTable({
+        items: rows,
+        itemHeadings: headings
+    })}
+    ${Footer()}
+
+
+    `;
+}
+
+function EvaluationTableHeader(options){
+    const headings = [];
+    options.headings.forEach(function(el){
```

Comment:
Because map iterates over an array and transforms each member of that array in the way that we specified. After that it returns a new array with the transformed members.